stack-8.2.1.yaml

```diff
@@ -0,0 +1,51 @@
+# Build setup for ghc-8.2.1.
+# TODO: remove this file after Stackage releases an LTS for this GHC version.
+# of GHC.
+setup-info:
+  ghc:
+    windows64:
```

Comment:
Removed; I copied it from other instructions, but as you point out it's not tested (and easy enough for other devs to add).